main.go

```diff
@@ -139,12 +139,15 @@ func runCommand(command string, args []string) (string, error) {
 }
 
 func retryCommand(command string, args []string) (out string, err error) {
-	for i := 1; i <= 3; i++ {
-		out, err = runCommand(command, args)
-		if out == "" && err != nil {
+	out, err = runCommand(command, args)
+	if out == "" && err != nil {
```

Comment:
Shouldn't inspecting only `err` be good enough? Or would it also retry for failed audit in that case?

Reply:
Exactly, if we inspect only the err, then a failed audit (with any vuln check level) would also trigger a retry.